packages/hr-document-retrieval/src/orgmetra_hr_document_retrieval/__init__.py

```
@@ -0,0 +1,29 @@
+    "DocumentArtifact",
+    "DocumentArtifactReader",
+    "DocumentMetadataResolver",
+    "DocumentRetrievalAuthorization",
+    "DocumentRetrievalAuthority",
+    "DocumentRetrievalQuery",
+    "DocumentRetrievalResult",
+    "DocumentRetrievalScope",
+    "HrDocumentRetrievalError",
+    "ImmutableRetrievalAuditWriter",
+    "retrieve_hr_document",
+]
```

Ruff (0.16.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)